tsfc/driver.py

```diff
@@ -56,6 +57,10 @@ def compile_form(form, prefix="form", parameters=None, interface=None):
     logger.info(GREEN % "compute_form_data finished in %g seconds.", time.time() - cpu_time)
 
     kernels = []
+    if coffee:
```

Comment:
This now ignores the incoming interface and should say:
```python
if interface is None:
   if coffee:
       ...
   else:
```

Reply:
Although that then means that we handle `interface` is None in two places. So perhaps move this logic to `compile_integral`

Reply:
I don't really understand this code. Maybe you had better do it.